qiita_ware/test/test_private_plugin.py

```diff
@@ -379,6 +387,46 @@ def test_complete_job(self):
         self.assertEqual(c_job.status, 'error')
         self.assertIn('No such file or directory', c_job.log.msg)
 
+    def test_submit_to_EBI(self):
+        # setting up test
+        from qiita_ware.test.test_ebi import FASTA_EXAMPLE
```

Comment:
Any reason why this import statement is here and not at the top?

Reply:
Cause I forgot! Moving to the top